### ISSUE TITLE
showselfclasswho issues

### ISSUE THREAD

**Issue by tuttlesBlacksmith** [svof/svof#619]
5020h, 4160m, 23185e, 18804w ex  jaguar  Austere   @ml oprompt cexkdb-
[Curing]: SCOUT
You begin scouting ahead for danger.
= Sl Furl - hp full,Hl: full,Wd NE@24kts,Cr/Sp SW@0,Sea Rough
5020h, 4160m, 23185e, 18804w e  jaguar  Austere   @ml oprompt cekdb-
(svof): Ready for combat! (test defences mode, took 25.5s)
5020h, 4160m, 23185e, 18804w e  jaguar  Austere   @ml oprompt cekdb-
= Sl Furl - hp full,Hl: full,Wd NE@24kts,Cr/Sp SW@0,Sea Rough (4.403s)
5020h, 4160m, 23185e, 18804w e  jaguar  Austere   @ml oprompt cekdb-
I'm sorry, I don't know what "showselfclasswho" does.
I'm sorry, I don't know what "showselfclasswho" does.
I'm sorry, I don't know what "showselfclasswho" does.
= Sl Furl - hp full,Hl: full,Wd NE@24kts,Cr/Sp SW@0,Sea Rough
5020h, 4160m, 23185e, 18804w e  jaguar  Austere   @ml oprompt cekdb-
You have recovered balance on all limbs.
= Sl Furl - hp full,Hl: full,Wd NE@24kts,Cr/Sp SW@0,Sea Rough (1.617s)
5020h, 4145m, 23185e, 18804w ex  jaguar  Austere   @ml oprompt cexkdb-(-15m, 0.3%) 
Silver sparks dance across the heavens as the dazzling orb of the sun rushes past Achaea's three
rings, the second brilliantly coruscating at its passing.
= Sl Furl - hp full,Hl: full,Wd NE@24kts,Cr/Sp SW@0,Sea Rough
5020h, 4205m, 23185e, 18858w ex  jaguar  Austere   @ml oprompt cexkdb-(+60m, 1.4%) vconfig releasechannel inclient
(svof): Will use the 'inclient' release channel for updates.
5020h, 4205m, 23185e, 18858w ex  jaguar  Austere   @ml oprompt cexkdb-l
The sky darkens with the onset of night. Various lines from the sails are drawn tightly around
winches here, the sturdy ropes vibrating ever so slightly as the wind blows past them. The waist-
high rails that run the length of the ship on each side join here precisely in the middle, intricate 
woodwork creating the illusion that the beams had grown together while they were alive. A large 
winch fitted with a long chain takes up quite a lot of space, the chain connected an the anchor that 
rests against the boat when not in use. Antlers raised, a majestic stag stares toward the horizon 
from the prow. Solidly connected to the deck, a ship's ballista lies ready here. Affixed to the 
deck, a large onager is constructed here. Firmly tied to the ship's deck, an Arcanian arm thrower 
awaits its next target. Greyish white ironwood comprises a sturdy munitions rack positioned nearby. 
There is a small group of lady's slipper roots here.
You see a single exit leading north.
= Sl Furl - hp full,Hl: full,Wd NE@31kts,Cr/Sp SW@0,Sea Stormy
5020h, 4295m, 23185e, 18903w ex  jaguar  Austere   @ml oprompt cexkdb-(+90m, 2.1%) ts
(svof): Won't keep shield up anymore.
I cannot fathom the meaning of "showselfclasswho".
I cannot fathom the meaning of "showselfclasswho".
I'm sorry, I don't know what "showselfclasswho" does.
= Sl Furl - hp full,Hl: full,Wd NE@31kts,Cr/Sp SW@0,Sea Stormy
5020h, 4280m, 23175e, 18927w ex  jaguar  Austere   @ml oprompt cexkdb-(-15m, 0.3%) ts
(svof): Will keep shield up.
I cannot fathom the meaning of "showselfclasswho".
I cannot fathom the meaning of "showselfclasswho".
I'm sorry, I don't know what "showselfclasswho" does.
= Sl Furl - hp full,Hl: full,Wd NE@31kts,Cr/Sp SW@0,Sea Stormy
5020h, 4300m, 23185e, 18939w ex  jaguar  Austere   @ml oprompt cexkdb-(+20m, 0.5%) 
Dusk steals silently over the horizon and crimson and ochre engulf the firmament, their seductive
tones beckoning the sun to its nightly repose.

I'm using mudlet to play a Sentinel in Achaea.  for some time now, maybe a month or two I would get this; I cannot fathom the meaning of "showselfclasswho" during various inputs to the game by me, such as manuall touching shield, inking tattoos, eating certain herbs, etc.  I was told I needed to upgrade from single class svof to the in-client svo, which I did. It helped somewhat but this kept coming back.  
DragonKnight, the ruiner found my problem today and asked me to write up an issue and this is my first ever issue.  
I had always had a command separator identified for years as "|".  Naturally I did not think this was an issue.  When DragonKnight had me check it, he posted this:

Basically, the broken trigger in class based svo set it to showsellfinclasswho
And that was saved in your config file
So, when you loaded up inclient
It was still set
When you looked at config the fixed trigger corrected the bad variable setting

**Comment by PhoenixCodes** [comment]
Perhaps a fresh install of in-client should check configs? I'm sure other configs from class-baased could cause issues places, too.

While we're at it, config releasechannel inclient
